client/solve.go

```diff
@@ -475,3 +475,33 @@ func parseCacheOptions(opt SolveOpt) (*cacheOptions, error) {
 	}
 	return &res, nil
 }
+
+func writeMetaDataFile(ex ExportEntry, exporterResponse map[string]string) error {
+	if ex.Type == ExporterOCI || ex.Type == ExporterImage {
+		if resultJsonFile, ok := ex.Attrs["metadata-file"]; ok {
```

Comment:
missing `metadata-file` fields in exporter?

Reply:
The `metadata-file` defined in buildctl as an option is used to save the image metadata to a json file. In some users' cases, the built image SHA256 should be exposed after build.